src/handlers/StakeRequestedHandler.ts

```diff
@@ -0,0 +1,43 @@
+import ContractEntityHandler from './ContractEntityHandler';
+import StakeRequest from '../models/StakeRequest';
+
+export default class StakeRequestedHandler extends ContractEntityHandler<StakeRequest> {
+  /**
+   * This method parse stakeRequest transaction and returns stakeRequest model object.
+   * @param transactions Transaction objects.
+   */
+  public parse = (transactions: any[]): StakeRequest[] => transactions.map((transaction) => {
+    const {
+      gasLimit,
+      gateway,
+      staker,
+      gasPrice,
+      id,
+      nonce,
+      beneficiary,
+      amount,
+      stakeRequestHash,
+    } = transaction;
+    return new StakeRequest(
+      id,
+      amount,
+      beneficiary,
+      gasPrice,
+      gasLimit,
+      nonce,
+      staker,
+      gateway,
+      stakeRequestHash,
+    );
+  });
+
+  /**
+   * This method defines action on receiving stake request model.
+   * @param stakeRequest instance of StakeRequest model .
+   */
+  public handle = (stakeRequest: StakeRequest[]): void => {
+
+    // stakeRequestRepository.save(stakeRequest);
```

Comment:
There is no save method in StakeRequestRepository. There is create method. Please see the updated StakeRequestRepository.ts code.

Reply:
It was an example comment 🤦‍♂